src/key.rs

```diff
@@ -85,7 +85,7 @@ impl RedisKey {
         let val = if self.is_null() {
             None
         } else {
-            hash_get_key(self.ctx, self.key_inner, field)
+            hash_mget_key(self.ctx, self.key_inner, &[field])?.pop().unwrap()
```

Comment:
If the field doesn't exist `hash_mget_key` returns `None` in the corresponding (in this case, only) entry, so this will return `None` as well.

(I'm assuming you asked because of the `.unwrap()`? That's there because `pop()` returns an `Option` in case the vector doesn't have an element to pop. Since in this case we know the vector is exactly 1 element in size it should always be `Some`.)

Reply:
> If the field doesn't exist `hash_mget_key` returns `None` in the corresponding (in this case, only) entry, so this will return `None` as well.

Good.
 
> (I'm assuming you asked because of the `.unwrap()`?

Indeed.

> That's there because `pop()` returns an `Option` in case the vector doesn't have an element to pop. Since in this case we know the vector is exactly 1 element in size it should always be `Some`.)

Very well; so please document that assumption in the code by using `expect(...)` with an appropriate message instead of `unwrap`.

Reply:
:+1: